app.js

```diff
@@ -10,6 +10,7 @@ const MODELS_DIR = 'models';
 const MODELS_MOUNT_PATH = '/models';
 const INDEX_PATH = path.join(__dirname + '/index.html');
 
+app.use(express.static('public'));
```

Comment:
3 things:
1. nit: plz pull literals into constants defined top of file.
2. what is public directory supposed to represent? what goes inside it vs. outside?
3. do we not need to change JS_DIR since we've moved it inside public?

I'm not 100% sure what best practices are for setting up assets directory with express so lmk if my questions are off mark :)

Reply:
By convention (at least regarding `express`), `/public` contains all the publically available files users will see (including the styles/script/images/... needed for the HTML pages) when using the web app.
A good example of in/out when it comes to express is:
![image](https://user-images.githubusercontent.com/8260834/77835845-ca124380-7148-11ea-911c-c36c016fc30a.png)
Source: https://expressjs.com/en/starter/generator.html

Since @amirhmk copied the `js/*.js` files to `public/js` and that those are used directly by `index.html`, the line with `app.use(JS_MOUNT_PATH, express.static(JS_DIR));` and both `JS_*` constants can be removed.

Reply:
Excellent thank you for the explanation!